src/AzSK.ADO/Framework/Abstracts/SVTCommandBase.ps1

```diff
@@ -101,16 +101,18 @@ class SVTCommandBase: CommandBase {
     [string] EvaluateControlStatus() {
         $startScan = ([CommandBase]$this).InvokeFunction($this.RunAllControls);
         if( ([FeatureFlightingManager]::GetFeatureStatus("EnableScanAfterAttestation","*"))) { 
-            if (($this.AttestationOptions.AttestControls -eq "NotAttested") -or ($this.AttestationOptions.AttestControls -eq "All")) {
-                if (Get-Variable AttestationValue -Scope Global){
-                    if ($Global:AttestationValue) {
-
-                        $this.PublishCustomMessage(([Constants]::DoubleDashLine))
-                        $this.PublishCustomMessage(([Constants]::HashLine))
-                        $this.PublishCustomMessage(([Constants]::AttestedControlsScanMsg))
-                        $this.PublishCustomMessage(([Constants]::DoubleDashLine))
-
-                        ([CommandBase]$this).InvokeFunction($this.ScanAttestedControls,$null);
+            if ($null -ne $this.AttestationOptions) {
```

Comment:
Why this condition?

Reply:
This block is to rescan in case of attestation. As we are not providing attestation in this command, $this.AttestationOptions is null. Added this condition to skip the rescanning